zookeeper-server/src/main/java/org/apache/zookeeper/server/ServerMetrics.java

```diff
@@ -74,6 +74,16 @@
     CONNECTION_TOKEN_DEFICIT(new AvgMinMaxCounter("connection_token_deficit")),
     CONNECTION_REJECTED(new SimpleCounter("connection_rejected")),
 
+    SYNC_PROCESSOR_QUEUE_TIME(new AvgMinMaxPercentileCounter("sync_processor_queue_time_ms")),
+    SYNC_PROCESSOR_QUEUE_SIZE(new AvgMinMaxCounter("sync_processor_queue_size")),
+    SYNC_PROCESSOR_QUEUED(new SimpleCounter("sync_processor_request_queued")),
+    SYNC_PROCESSOR_REAL_QUEUE_TIME(new AvgMinMaxPercentileCounter("sync_processor_real_queue_time_ms")),
```

Comment:
Let's remove this one, and change SYNC_PROCESSOR_QUEUE_TIME to track the real queue time.

Reply:
done